exericse4/exercise2.js

```diff
@@ -0,0 +1,10 @@
+function isPrimeNumber(number) {
+  if (number < 2) return false;
+  if (number === 2) return true;
```

Comment:
I think this additional checking **if (number ===  2) return true** is not necessary as long you have for loop, which is also checking again for equality of numbers with 2